patterns/README.md

```diff
@@ -8,6 +8,36 @@ Welcome to the repository! Below is a list of available documents and workflows
 
 - [Lint and Test Workflow](github-workflows/lint-test.yaml)
 - [Release Workflow](github-workflows/release.yml)
+- [Compatibility Testing Workflow](github-workflows/compatibility-testing-example.yaml)
+
+#### How to setup Compatibility Testing in GitHub Actions
+
+This tutorial will guide you through setting up compatibility testing in GitHub Actions for your application. It will use the Replicated Replicated Compatibility Matrix to test your application against different versions of Kubernetes and different operating systems.
```

Comment:
```suggestion
This tutorial will guide you through setting up compatibility testing in GitHub Actions for your application. It will use the Replicated Compatibility Matrix to test your application against different versions of Kubernetes and different operating systems.
```

Reply:
Thank you @jdewinne!